lib/rules/jsx-no-literals.js

```diff
@@ -25,27 +30,38 @@ module.exports = {
 
   create: function(context) {
 
+    const isNoStrings = context.options[0] === NO_STRINGS;
+
+    const message = isNoStrings ?
+      'Strings not allowed in JSX files' :
+      'Missing JSX expression container around literal string';
+
     function reportLiteralNode(node) {
       context.report({
         node: node,
-        message: 'Missing JSX expression container around literal string'
+        message: message
       });
     }
 
+    function getValidation(node) {
+      const standard = !/^[\s]+$/.test(node.value) &&
+          node.parent &&
+          node.parent.type.indexOf('JSX') !== -1 &&
```

Comment:
This was the existing validation. I just removed part of it if you enable the option.